### PR TITLE
Add dockerfile for jsonpath-web-validator

### PR DESCRIPTION
**Pull request by ikalinin1**

### CODE REVIEW

**Review by valfirst — APPROVED** [review]

File: .github/workflows/docker.yml [hunk 1]
```diff
@@ -0,0 +1,44 @@
+name: Validate Jsonpath WEB validator Dockerfile
```
Comment (uarlouski) [inline .github/workflows/docker.yml]: maybe Jsonpath WEB validator deploy ?

File: .github/workflows/docker.yml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+          dockerfile: 'Dockerfile'
+          lint: 'hadolint'
+
+      - name: Set up Docker Buildx
+        id: buildx
+        uses: docker/setup-buildx-action@v2
+
+      - name: Build image
+        uses: docker/build-push-action@v4
+        id: build
```
Comment (uarlouski) [inline .github/workflows/docker.yml]: why do we need ids for Set up Docker Buildx and Build image?

File: .github/workflows/docker.yml (hunk 1 above)
Comment (ikalinin1) [inline .github/workflows/docker.yml]:
```suggestion
name: Deploy Jsonpath WEB
```

**Review by valfirst — APPROVED** [review]

File: .github/workflows/docker.yml [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
```
Comment (ikalinin1) [inline .github/workflows/docker.yml]:
```suggestion
```

**Review by uarlouski — APPROVED** [review]

**Pull request merged**